dll/win32/shell32/shellmenu/CStartMenu.cpp

```diff
@@ -167,7 +196,6 @@ class CShellMenuCallback :
         {
         case IDM_PROGRAMS:  csidl = CSIDL_PROGRAMS; break;
         case IDM_FAVORITES: csidl = CSIDL_FAVORITES; break;
-        case IDM_DOCUMENTS: csidl = CSIDL_RECENT; break;
```

Comment:
this is a hack. restore this please

Reply:
when csidl is CSIDL_RECENT we need to use IShellMenu::SetMenu  to add the static part on top of the dynamic part.

Reply:
Where is `IShellMenu::SetMenu` impl?

Reply:
OK.